### PR TITLE
Added gitignore, Linter, README, HTML, CSS

### PR DESCRIPTION
**Pull request by Kakalanp**
### **Final pull:**

- Adding a Git ignore to filter unnecessary files.
- Adding the Linter as demonstrated and provided by this exercise indications.
- Adding a Professional README file.
- Adding the HTML file containing "Hello Microverse" as specified by this exercise indications.
- Adding a CSS file which serves the purpose of coloring the above-mentioned text "Hello Microverse" to red.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md*
```
Comment (MrBrN197) [inline .gitignore]: - [x] The asterix (`*`) after the "test.md" is not necessary that will prevent any files that start with test.md we only want to ignore the test.md files not files that start with test.md

Comment (MrBrN197) [inline .gitignore]: - [x] Please add the /node_modules folder to the gitignore file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Hello Microverse Project
+
```
Comment (MrBrN197) [inline README.md]: - [x] It's important to create a more professional README file this is important as it provides details about your project.
You can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)  to create your README file.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @Kakalanp 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Please create a more descriptive pull request description. It's important to let others know what the purpose of the pull request is and what changes it introduces.
- [x] remember to check the inline comments below for more required changes

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
I don't know if the new description serves it's propose.

What should I do in case everything seems in order under my criteria (for another review)?

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Kakalanp , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- _It's important to create a more professional README file this is important as it provides details about your project you can use this [template](https://github.com/microverseinc/readme-template)._
- _Please create a more descriptive pull request description as requested by the previous reviewer. It's important to let others know what the purpose of the pull request is and what changes it introduces. Kindly check Example of good comments [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
> * It's important to create a more professional README file this is important as it provides details about your project you can use this [template](https://github.com/microverseinc/readme-template).

I updated the README file following that exact template as a reference, can you please articulate more on the problem behind this file, or how could I solve the problem? Thank you in advance.

**Review by eliasecasta — COMMENTED** [review]
Hello @Kakalanp 👋,

_This is an additional review as a result of the second opinion request._

Your PR is descriptive enough, this means that the title and description of it are good to go.

What you are missing is only to add a little more detail to your Readme file, I will link you an example of one of my Readmes so you can check out what are you missing, in this case, the only mandatory requirement that is not present is the installation instructions for the end-user because maybe someone that is not tech-savvy enough wants to use your application but doesn't know how to: https://github.com/e71az/newsweek/blob/master/README.md

As you can see in one of my Readme sections, I included the getting started section which you can appreciate here:

image

Please add something similar to help people run your projects in the future, after this, please ask for another review, it should be approved without issues, thank you and have a nice day/night 👍 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Comment by Kakalanp** [comment]
Thank you very much! I wasn't aware of that part being mandatory, else I would've added it from the beginning! I'm confident my project is finally ready, and I'm going to submit it right away, as you suggested. 🎉 

Again, thanks a lot for being that descriptive! Have a nice day/night, too. 👍

**Review by Whoistolu — APPROVED** [review]
### Hi @Kakalanp 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**